.github/CODEOWNERS

```diff
@@ -1 +1,41 @@
-* @rwalker-apple @chrisdecenzo @hawk248 @andy31415 @robszewczyk @bzbarsky-apple @BroderickCarlin @saurabhst @jelderton @mspang
+# Each line is a file pattern followed by one or more owners.
+
+# Current reviewers-XXX members.  Review of *every* PR must pass through
+#  these folks in order for "pull-approve" CI to pass
```

Comment:
nit: 👾

Reply:
help.  I don't understand " 👾" "

Reply:
simply intended to indicate you have extra space(s) on this line (aka space invaders 😄)

Reply:
ah.  that was on purpose, actually.  I thought you might have been referring to the byzantine language, which I trimmed.